### DISCUSSION TITLE
ScriptReference/SystemInfo-graphicsMemorySize

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#250] [Notes]
# ScriptReference/SystemInfo-graphicsMemorySize

https://docs.unity3d.com/ScriptReference/SystemInfo-graphicsMemorySize.html

**Comment by radiatoryang** [comment]
Be warned that this is a pretty useless stat in practice; I think Windows will automatically allocate more memory for onboard GPUs.
